gapis/api/vulkan/vulkan.api

```diff
@@ -193,6 +201,9 @@ sub ref!ExtensionSet supportedInstanceExtensions() {
   supported.ExtensionNames["VK_KHR_win32_surface"] = true
   supported.ExtensionNames["VK_KHR_get_physical_device_properties2"] = true
   supported.ExtensionNames["Vk_KHR_get_surface_capabilities2"] = true
```

Comment:
I think these might be wrong?

V**k**_KHR_external

Reply:
Fixed in 68a52c2